frontend/src/metabase/query_builder/components/view/ViewHeader/ViewHeader.jsx

```diff
@@ -256,13 +257,8 @@ function SavedQuestionLeftSide(props) {
           <HeadBreadcrumbs
             divider={<HeaderDivider>/</HeaderDivider>}
             parts={[
-              ...(isAdditionalInfoVisible && isDataset
-                ? [
-                    <DatasetCollectionBadge
-                      key="collection"
-                      dataset={question}
-                    />,
-                  ]
+              ...(isAdditionalInfoVisible && type !== "question"
```

Comment:
`question.type()` can be `undefined`, so I think we should be checking both `metric` and `model`:

```suggestion
              ...(isAdditionalInfoVisible && isMetricOrModel
```

Or we can `const type = question.type() ?? "question";`

Reply:
@kamilmielnik 
Done in https://github.com/metabase/metabase/pull/38554/files#diff-2d5a41263f669d5990aeda6eff70f7b87260d4070ac63784f0ed86a4aea8048aR227, but I used `||`. Do you mind that? It's slightly more permissive in a sense that the fallback happens for all falsy values, not just `null` or `undefined`.

They should logically work the same in this scenario.